api/src/main/java/org/jboss/forge/roaster/model/util/Types.java

```diff
@@ -153,8 +144,29 @@
 
    private Types()
    {
+      // Util class
```

Comment:
```throw new IllegalAccessError("Utility class");```

Reply:
If it is private, it can't be called, so no exception is needed. Or do want to catch the reflection case?

Reply:
Yes, the reflection case :)

Reply:
okay, let me add this